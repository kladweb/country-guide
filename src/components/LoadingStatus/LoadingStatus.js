export const LoadingStatus = ({loadStatus}) => {
  return (
    <>
      <div className='loading'>
         <span className='loading-status'>{loadStatus}</span>
      </div>
    </>
  )
}
