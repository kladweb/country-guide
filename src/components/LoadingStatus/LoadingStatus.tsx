import React from "react";

interface ILoadingStatusProps {
  loadStatus: string
}

export const LoadingStatus: React.FC<ILoadingStatusProps> = ({loadStatus}) => {
  return (
    <>
      <div className='loading'>
        <span className='loading-status'>{loadStatus}</span>
      </div>
    </>
  )
}
