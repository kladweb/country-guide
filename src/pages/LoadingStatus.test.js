import React from 'react';

import renderer from 'react-test-renderer';

import { LoadingStatus } from "./LoadingStatus.js";

test('работа LoadingStatus', () => {

  // создаём тестовую версию компонента
  const component = renderer.create(
    <LoadingStatus/>
  );

  let componentTree = component.toJSON();
  expect(componentTree).toMatchSnapshot();

});