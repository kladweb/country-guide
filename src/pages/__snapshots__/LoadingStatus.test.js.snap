// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа LoadingStatus 1`] = `
<div
  className="loading"
>
  <span
    className="loading-status"
  />
</div>
`;
